### PR TITLE
fix: model init ValueError

### LINKED ISSUE #13438: model init ValueError
### System Info

langchain version: 0.0.336
mac
python3.8

### Who can help?

_No response_

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

```py
import langchain
from langchain.chat_models.minimax import MiniMaxChat
print(langchain.__version__)
chat = MiniMaxChat(minimax_api_host="test_host", minimax_api_key="test_api_key", minimax_group_id="test_group_id")
assert chat._client
assert chat._client.host == "test_host"
assert chat._client.group_id == "test_group_id"
assert chat._client.api_key == "test_api_key"
```

output:

```sh
0.0.336
Traceback (most recent call last):
  File "/Users/hulk/code/py/workhome/test_pydantic/test_langchain.py", line 4, in <module>
    chat = MiniMaxChat(minimax_api_host="test_host", minimax_api_key="test_api_key", minimax_group_id="test_group_id")
  File "/Users/hulk/miniforge3/envs/py38/lib/python3.8/site-packages/langchain/llms/minimax.py", line 121, in __init__
    self._client = _MinimaxEndpointClient(
  File "pydantic/main.py", line 357, in pydantic.main.BaseModel.__setattr__
ValueError: "MiniMaxChat" object has no field "_client"
```

### Expected behavior

Instantiation Success

### PR DESCRIPTION
**Pull request by vvanglro**
This pr fix #13438 #11648 #10917

### CODE REVIEW

**Comment by baskaryan** [comment]
cc @hetaoBackend

**Comment by vvanglro** [comment]
@baskaryan Please run ci again.

**Comment by vvanglro** [comment]
Hi@baskaryan.
I investigated the entire revision history, The cause of the error occurred here #10776. 
This commit had a fix attempted but didn't find the problem #12554 .

**Comment by hetaoBackend** [comment]
Sorry for the bug, Thanks for fixing it.

**Comment by vvanglro** [comment]
@baskaryan Please review.

File: libs/langchain/langchain/llms/minimax.py [hunk 1]
```diff
@@ -87,6 +87,11 @@ def validate_environment(cls, values: Dict) -> Dict:
             "MINIMAX_API_HOST",
             default="https://api.minimax.chat",
         )
+        cls._client = _MinimaxEndpointClient(
```
Comment (hwchase17) [inline libs/langchain/langchain/llms/minimax.py:90]: i think slightly better to do 
```suggestion
        values["_client"] = _MinimaxEndpointClient(
```

Comment (vvanglro) [reply libs/langchain/langchain/llms/minimax.py]: agree

**Comment by baskaryan** [comment]
looks like this has been fixed, ok to close @vvanglro!

**Pull request closed without merging**